server.js

```diff
@@ -18,8 +20,32 @@ import env from "./config/env.generated.json";
 const defaultPort = 3000;
 const PORT = env.PORT || process.env.PORT || defaultPort;
 
+/**
+ * Some app security settings
+ */
+
```

Comment:
this is function comment syntax (at least as per jsdoc/javadoc) so you probably want this to be `// Some app security settings` since this comment doesn't actually "document" anything. And then have it tied to the next line without a newline in between